corehq/apps/userreports/README.md

```diff
@@ -616,6 +616,23 @@ numeric_percent | Percentage as a number                         | 33
 decimal         | Fraction as a decimal number                   | .333
 
 
+### The "aggregation" column property
+
+The aggregation column property defines how the column should be aggregated.
+If the report is not doing any aggregation, or if the column is one of the aggregation columns this should always be `"simple"` (see [Aggregation](#aggregation) below for more information on aggregation).
+
+The following table documents the other aggregation options, which can be used in aggregate reports.
+
+Format          | Description
+--------------- | -----------------------------------------------
+simple          | No aggregation
```

Comment:
I can never seem to remember: Do columns with `"aggregation": "simple"` get an arbitrary value from the grouped rows, or is that an error?